e2e/regression/guardrail.correction_range.spec.js

```diff
@@ -0,0 +1,50 @@
+const { LoopTest, screenName, target } = require('../../src/index');
+
+describe.skip('guard rail correction range', () => {
+    var loopTest;
+    it('should setup with correct configuration', async () => {
+        loopTest = await new LoopTest.Builder(target.tidepool)
```

Comment:
[nit] Consider something like:
```
test = new Test().withSettings({ AddPumpSimulator: true})
                              .withStartScreen(screenNames.settings);
await test.prepare();
```
Note: 
  - `loopTest` -> `test`
  - `LoopTest` -> `Test`
  - no separate `Builder` object
  - move `target.tidepool` to top-level test-run construction so it can be configured outside of the test framework
  - `build` -> `prepare` (as in preparing the test)